<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleSettings</class>
 <widget class="QDialog" name="VehicleSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Impostazioni Veicolo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/images/icons/vehicle.png</normaloff>:/images/icons/vehicle.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QComboBox" name="vehicleCombo">
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContents</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="addButton">
      <property name="text">
       <string>Aggiungi</string>
      </property>
      <property name="icon">
       <iconset resource="../mapgraphics/resources.qrc">
        <normaloff>:/icons/images/edit_add.png</normaloff>:/icons/images/edit_add.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="removeButton">
      <property name="text">
       <string>Rimuovi</string>
      </property>
      <property name="icon">
       <iconset resource="../mapgraphics/resources.qrc">
        <normaloff>:/icons/images/editdelete.png</normaloff>:/icons/images/editdelete.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Veicolo</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vehicleName"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Modello:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vehicleModel"/>
      </item>
      <item>
       <widget class="QToolButton" name="vehicleModelButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Descrizione:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="vehicleDesc"/>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Salva</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="vehicleImgPath">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>4</y>
     <width>298</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Immagine</string>
   </property>
   <widget class="QLabel" name="vehicleImg">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>288</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../mapgraphics/resources.qrc"/>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
